manifests/repo.pp

```diff
@@ -32,15 +32,14 @@
 
   case $::osfamily {
     'Debian': {
-      if !defined(Class['apt']) {
-        class { 'apt': }
-      }
+      include ::apt
+      Class['apt::update'] -> Package<| |>
 
       apt::source { 'elasticsearch':
         location    => "http://packages.elastic.co/elasticsearch/${elasticsearch::repo_version}/debian",
         release     => 'stable',
         repos       => 'main',
-        key         => 'D88E42B4',
+        key         => '46095ACC8548582C1A2699A9D27D666CD88E42B4',
```

Comment:
PE 3.3.2 has puppetlabs-apt 1.5.0 included so can't upgrade that and it supports only the short notations.
Unless i drop 3.3.2 support ( which i most likely will do before 1.0 release ) the key will have to be the short version.
If you could change that i can merge this and hopefully solve the apt 2.x issue :-)